spec/features/publish/viewing_a_course_2025_spec.rb

```diff
@@ -1,317 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails_helper'
```

Comment:
Same as above, just want to make sure that we don't still need these tests but without the year constraint

Reply:
All of the tests other than for personal qualities and other requirements are tested in here: 

spec/features/publish/viewing_a_course_spec.rb

i just renamed it because its not year specific anymore.

Reply:
That makes sense, great!